vrt-install.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+
+DOCKER_URL=https://docs.docker.com/get-docker/
+DOCKER_COMPOSE_URL=https://docs.docker.com/compose/install/
+
+DOCKER_YML_URL=https://raw.githubusercontent.com/Visual-Regression-Tracker/Visual-Regression-Tracker/master/docker-compose.yml 
+DOCKER_ENV_URL=https://raw.githubusercontent.com/Visual-Regression-Tracker/Visual-Regression-Tracker/master/.env
+ENV_FILE=.env
+
+REACT_APP_API_URL=
+APP_FRONTEND_URL=
+IMG_UPLOAD_FOLDER=
+POSTGRES_FOLDER=
+JWT_SECRET=`cat /dev/urandom | env LC_CTYPE=C tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1`
+
+usage()
+{
+    cat <<EOF
+Installs the Visual Regression Tracker
+
+Usage: ./vrt-install.sh
+
+Arguments:
+    -h | --help
+    -a | --frontend-url <url>   Set the Front-end url. Default: http://localhost:8080
+    -r | --backend-url <url>    Set the API url. Default: http://localhost:4200
+    -i | --images-folder <dir>  Set the location of the image folder.
```

Comment:
please, remove this as param 
found that it works not as I intended and value from `env` is defining folder name inside container 
we gonna use `imageUploads` as hardcoded value and I'll remove this from `env` file

Reply:
will do it in scope of 
https://github.com/Visual-Regression-Tracker/Visual-Regression-Tracker/issues/160

Reply:
👍